app/components/ui/GraphCard.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 type Props = React.PropsWithChildren<{
   title: string;
-  subtitle?: string;
+  subtitle?: string | React.ReactNode;
```

Comment:
```suggestion
  subtitle?: React.ReactNode;
```

Checkout the type definition of `React.ReactNode` and see if this throws any errors. I'm pretty sure that this would be valid and correct.

Reply:
Yep good catch, worked perfectly :)